analyses/copy_number_consensus_call/Snakefile

```diff
@@ -2,9 +2,9 @@
 # Updated Dec 5, 2019
 
 ## Define the ending file(s) that we want
-ALL_FREEC= expand("../../scratch/interim/{sample}.freec.dup.bed", sample=config["samples"])
-ALL_CNVKIT= expand("../../scratch/interim/{sample}.cnvkit.dup.bed", sample=config["samples"])
-ALL_MANTA= expand("../../scratch/interim/{sample}.manta.dup.bed", sample=config["samples"])
+ALL_FREEC= expand("../../scratch/interim/{sample}.freec.dup.filtered.bed", sample=config["samples"])
+ALL_CNVKIT= expand("../../scratch/interim/{sample}.cnvkit.dup.filtered.bed", sample=config["samples"])
+ALL_MANTA= expand("../../scratch/interim/{sample}.manta.dup.filtered.bed", sample=config["samples"])
```

Comment:
Similarly to what I did in the pull request I filed, we can simplify this a bit, taking advantage of snakemake features, and reduce redundancy: 

```suggestion
OUTPUT= expand("../../scratch/interim/{sample}.{caller}.dup.filtered.bed", 
               sample=config["samples"], 
               caller=["freec", "cnvkit", "manta"])
```

Reply:
I never knew that you could do this with Snakemake. I had so many questions when I was learning Snakemake. I am glad you are showing me this now. This DOES reduce so much redundancy. Thanks you :)